### DISCUSSION TITLE
Aplikacja do mierzenia wydajnosci mocy obliczeniowej Azure

### DISCUSSION THREAD

**Discussion by matslawski** [AzureCommunityPL/group#188] [Pomoc]
Czesc,

Mam do stworzenia na studiach aplikacje mierzącą wydajność mocy obliczeniowej chmury. Pomyslalem wiec ze najprosciej bedzie pobrac dane poprzez API. Niestety ilosc wszelkich informacji troche mnie przygniotla i prosilbym o jakies porady jak najlepiej korzystac w takim przypadku z Azure API

**Comment by kamil-mrzyglod** [comment]
Ale co masz dokładnie na myśli? Ilość vCore? Limity per subskrypcja? Jakie dokładnie dane Ciebie przerosły? Jak je pobrałeś?

**Reply by smereczynski** [reply]
Albo kolega źle przekazał nam zadanie, albo zadający zadanie nie ma pojęcia o tym, czego oczekuje od studentów :)

**Reply by wi5nia** [reply]
Bo to jest zadanie podchwytliwe. Moc obliczenia chmury jest w zasadzie nieograniczona!
